code/edgemedia/src/test/java/com/adobe/marketing/mobile/edge/media/internal/MediaEventTrackerTests.java

```diff
@@ -115,8 +124,39 @@ public MediaEventTrackerTests() {
 
         mediaTrackerAPIEventGenertor = TestMediaPublicTracker.create("tracker0", true);
 
-        hitProcessor = new FakeMediaHitProcessor();
-        tracker = new MediaEventTracker(hitProcessor, config);
+        mockEventProcessor = Mockito.mock(MediaEventProcessor.class);
+        mockSessionMap = new HashMap<>();
+        tracker = new MediaEventTracker(mockEventProcessor, config);
+
+        setMocks();
+    }
+
+    private void setMocks() {
```

Comment:
nit: Is there a reason why when calling `createSession` you use `Mockito.when().thenAnswer()` while for `processEvent` you use `Mockito.doAnswer().when()`?  I've read you must you `doAnswer` when stubbing a method which returns Void (like in processEvent), but I'm wondering for consistency can `doAnswer` be used for stubbing createSession as well?